<?xml version="1.0" encoding="utf-8" standalone="no"?>
<!--This file represents the results of running a test suite-->
<test-results name="POMExample.dll" total="2" errors="0" failures="1" not-run="0" inconclusive="0" ignored="0" skipped="0" invalid="0" date="2019-03-26" time="16:38:07">
  <environment nunit-version="3.6.0.0" clr-version="4.0.30319.42000" os-version="Microsoft Windows NT 10.0.16299.0" platform="Win32NT" cwd="C:\Program Files (x86)\Jenkins\workspace\nunit example" machine-name="GCND8251LFTE" user="SYSTEM" user-domain="LOGON" />
  <culture-info current-culture="en-US" current-uiculture="en-US" />
  <test-suite type="Assembly" name="POMExample.dll" executed="True" result="Failure" success="False" time="230.870" asserts="8">
    <properties>
      <property name="_PID" value="18356" />
      <property name="_APPDOMAIN" value="domain-" />
    </properties>
    <failure>
      <message><![CDATA[One or more child tests had errors]]></message>
      <stack-trace />
    </failure>
    <results>
      <test-suite type="TestSuite" name="POMExample" executed="True" result="Failure" success="False" time="230.852" asserts="8">
        <failure>
          <message><![CDATA[One or more child tests had errors]]></message>
          <stack-trace />
        </failure>
        <results>
          <test-suite type="TestFixture" name="TestClass" executed="True" result="Success" success="True" time="90.098" asserts="4">
            <results>
              <test-case name="POMExample.TestClass.SearchTextFormAboutPage" executed="True" result="Success" success="True" time="87.188" asserts="4">
                <properties>
                  <property name="Order" value="1" />
                </properties>
              </test-case>
            </results>
          </test-suite>
          <test-suite type="TestFixture" name="TestClass2" executed="True" result="Failure" success="False" time="140.748" asserts="4">
            <failure>
              <message><![CDATA[One or more child tests had errors]]></message>
              <stack-trace />
            </failure>
            <results>
              <test-case name="POMExample.TestClass2.SearchTextFormAboutPage2" executed="True" result="Failure" success="False" time="140.733" asserts="4">
                <properties>
                  <property name="Order" value="1" />
                </properties>
                <failure>
                  <message><![CDATA[  Expected: greater than 100
  But was:  10
]]></message>
                  <stack-trace><![CDATA[   at POMExample.TestClass2.SearchTextFormAboutPage2() in C:\mycode\dotnet\SeleniumInCSharp\POMExample\POMExample\TestCases\TestClass2.cs:line 104
]]></stack-trace>
                </failure>
              </test-case>
            </results>
          </test-suite>
        </results>
      </test-suite>
    </results>
  </test-suite>
</test-results>